### PR TITLE
Add tailwindcss and all relevant stylings

### PR DESCRIPTION
**Pull request by sechaba-mache**

### CODE REVIEW

File: tailwind.config.cjs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ["./*.{html, js}"],
+	theme: {
+		colors: {
+			aliceblue: "#f0f8ff",
```
Comment (CharelMarais) [inline tailwind.config.cjs:6]: Nice, custom colors and values.

File: tailwind.config.cjs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ["./*.{html, js}"],
```
Comment (CharelMarais) [inline tailwind.config.cjs:3]: Add ts to the list `{html,js}` when you transfer the work to ts

File: style.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	}
+}
+
+.error {
+	display: flex;
+	justify-content: center;
+	width: 100vw;
+}
+
+.box {
```
Comment (CharelMarais) [inline style.css:24]: You can use classList.add() or className to add tailwind to your dom-manip file or with your back-tics just like the normal HTML tailwind usage.

**Review by CharelMarais — CHANGES_REQUESTED** [review]
Looks good but you can look at implementing tailwind in your dom-manipulation js file as well.

File: tailwind.config.cjs (hunk 2 above)
Comment (sechaba-mache) [reply tailwind.config.cjs:3]: Will do

**Comment by sechaba-mache** [comment]
> Looks good but you can look at implementing tailwind in your dom-manipulation js file as well.

I was thinking about doing that but it would make the dom-manip file too clustered which is why i decided to just implement the styling in the css.

**Review by CharelMarais — APPROVED** [review]
If that is the case you are good to go.

**Comment by sechaba-mache** [comment]
> If that is the case you are good to go.

Arigato gozaimasu

**Pull request merged**